dotCMS/src/main/java/com/dotcms/rest/api/v1/site/SiteBrowserHelper.java

```diff
@@ -95,40 +97,71 @@ public String getHostManagerUrl(final HttpServletRequest req, final List<Layout>
         return null;
     } // getHostManagerUrl.
 
-    /**
-     * Return the list of host that the given user has permissions to see.
-     *
-     * @param showArchived if it is false the archived host aren't returned
-     * @param user
-     * @param filter if it is not a empty String then just the hosts with a hostName starting by 'filter' are returned
-     * @return List of host that the given user has permissions to see and filter by filter
-     * @throws DotDataException if one is thrown when the sites are search
-     * @throws DotSecurityException if one is thrown when the sites are search
-     */
-    public List<Host> getOrderedHost(boolean showArchived, User user, String filter) throws DotDataException, DotSecurityException {
-        return this.hostAPI.findAll(user, Boolean.TRUE)
+	/**
+	 * Returns the list of sites that the given user has access to.
+	 *
+	 * @param showArchived
+	 *            - Is set to {@code true}, archived sites will be returned.
+	 *            Otherwise, set to {@code false}.
+	 * @param user
+	 *            - The {@link User} performing this action.
+	 * @param filter
+	 *            - (Optional) If specified, returns the sites whose name starts
+	 *            with the value of the {@code filter} variable.
+	 * @return The list of sites that the given user has permissions to access.
+	 * @throws DotDataException
+	 *             An error occurred when retrieving the sites' data.
+	 * @throws DotSecurityException
+	 *             A system error occurred.
+	 */
+    public List<Host> getOrderedSites(final boolean showArchived, final User user, final String filter) throws DotDataException, DotSecurityException {
```

Comment:
I have talked this with @freddyucv before. Aren't these methods good candidates to be in the HostAPI?